org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/hyperlinking/SingleHoverShowingHyperlinkPresenter.xtend

```diff
@@ -79,7 +79,7 @@ class SingleHoverShowingHyperlinkPresenter implements InvocationHandler {
 				if (it !== null && it.hyperlinkRegion !== null) {
 					list.add(it)
 				} else {
-					log.warn('Filtered invalid hyperlink: '+ it.getClass.name)
+					log.warn('Filtered invalid hyperlink: '+ it?.class?.name)
```

Comment:
Why not just a logging of the value of it?

Reply:
cant tell why its done this way.